tests/desktop-e2e/scripts/test-round-trip.ps1

```
@@ -39,12 +42,48 @@ function Test-Fail {
     Write-Host "FAIL: $Name"
 }
 
+function Ensure-VerifierRuntime {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Ensure-VerifierRuntime' uses an unapproved verb.

(PSUseApprovedVerbs)